frontend/public/locales/en/gcweb.json

```diff
@@ -67,5 +67,11 @@
     "header_one": "The following error was found in the form:",
     "header_other": "The following {{count}} errors were found in the form:"
   },
-  "asterisk-indicates-required-field": "An asterisk (*) indicates a required field"
+  "asterisk-indicates-required-field": "An asterisk (*) indicates a required field",
+  "session-timeout": {
+    "continue-session": "Continue session",
+    "description": "Your session will expire automatically in {{timeRemaining}}. Select \"Continue Session\" to extend your session.",
+    "end-session": "End session now",
+    "header": "Session timeout warning"
```

Comment:
I think the dialog headline should be `Extend your session?` based on Material Design Dialog specs.

https://m3.material.io/components/dialogs/guidelines#0cff8b6b-d4f3-4442-8194-31212f5d1a12

@gregory-j-baker any comment?

Reply:
We can refactor later.